docs/changelog.rst

```diff
@@ -131,10 +131,10 @@ Features - 4.0.0a7
 - Implement ``[testenv] ignore_outcome`` - "a failing result of this testenv will not make tox fail" - by :user:`hexagonrecursion`. (`#1947 <https://github.com/tox-dev/tox/issues/1947>`_)
 - Inline plugin support via ``tox_.py``. This is loaded where the tox config source is discovered. It's a Python file
   that can contain arbitrary Python code, such as definition of a plugin. Eventually we'll add a plugin that allows
-  succinct declaration/generation of new tox environments - by :user:`gaborbernat`. (`#1963 <https://github.com/tox-dev/tox/issues/1963>`_)
+  succinct declaration/generation of new tox environments - by :user:`gaborbernat`. (`#1963 <https://github.com/tox-dev/tox/pull/1963>`_)
 - Introduce the installer concept, and collect pip installation into a ``pip`` package, also attach to this
-  the requirements file parsing which got a major rework - by :user:`gaborbernat`. (`#1991 <https://github.com/tox-dev/tox/issues/1991>`_)
-- Support CPython ``3.10`` -by :user:`gaborbernat`. (`#2014 <https://github.com/tox-dev/tox/issues/2014>`_)
+  the requirements file parsing which got a major rework - by :user:`gaborbernat`. (`feature #1991 <https://github.com/tox-dev/tox/pull/1991>`_)
```

Comment:
One more addition of the word `feature`.

Reply:
Same as above.